test/cppsuite/README.md

```diff
@@ -1 +1,132 @@
+
+## Update
+An update operation uses a random cursor to select a key in the database and updates it with a random value.
+The number of update threads, updates performed in each transaction, sleep time between each update, size of keys and values are defined in the configuration file.
+
+## Remove
+A remove operation uses a random cursor to select and remove a key from the database.
+The number of remove threads, removals performed in each transaction and sleep time between each removal are defined in the configuration file.
+
+## Read
+A read operation selects a random collection and uses a random cursor to read items from it.
+The number of read threads, reads performed in each transaction and the sleep time between each read are defined in the configuration file.
+
+## Custom
+There is no default implementation as this function is intended to be user-defined.
+The number of threads, operations performed in each transaction, sleep time between each operation, size of the keys and values are defined in the configuration file. 
+
+## Checkpoint
+A checkpoint operation executes a checkpoint on the database every 60 seconds.
+Checkpoints are enabled or disabled by assigning one or zero threads in the configuration file, they are enabled by default. Checkpoints are executed periodically according to the configured operation rate.
+
+## Validate
+The default validation algorithm requires the default [operation tracker](#operation-tracker) configuration. If the operation tracker is reconfigured, the default validation is skipped.
+The default validator checks if the WiredTiger tables are consistent with the tables tracked by the operation tracker by comparing their content. The validation (and hence the test) fails if there is any mismatch.
+
+# Components
+The framework provides built-in components that each offer a set of features to enhance the testing experience. Their behaviour is customized through a configuration file.  A component has a life cycle made of three stages: *load*, *run*, and *finish*. Each of these stages is described for each component below.
+
+
+## Workload Manager
+The workload manager is responsible for calling the populate function and the lifecycle of the threads dedicated to each of the database operations described above.
+
+| Phase    | Description |
+| -------- | ----------- |
+| Load     | N/A         |
+| Run      | Calls the populate function, then parses the configuration for each operation and spawns threads for each of them. |
+| Finish   | Ends each thread started in the run phase. |
+
+## Operation Tracker
+During the execution of the test, by default the operation tracker saves test metadata every time a thread performs an update, insertion, or removal operation. The user can also manually save test metadata by calling the `save_operation` function from the `operation_tracker` class.
```

Comment:
We can drop "by default" here now we mention manually saving data in the next sentence

Reply:
Fixed in 676c021ff429c8b259dc76bdf53cb7948d74edaa